portal/eproms_substudy_tailored_content/src/js/Base.js

```diff
@@ -428,6 +428,14 @@ export default {
                     .then(() => {
                         // DOM updated
                         this.onDomainContentDidLoad();
+                        if (this.isAtMainPage(this.getSelectedDomain())) {
+                            return;
+                        }
+                        //log accessed content domain url 
+                        postData("/api/auditlog", {
+                            "message": "GET " + window.location.pathname + window.location.hash,
```

Comment:
should there be a delimiter (say `/`) between pathname and location?

Reply:
Thank you for noticing that. The delimiter is  included in the variable, window.location.pathname.